docs/SetupDatabaseForAWS.md

```diff
@@ -1,30 +1,151 @@
-# Set up a database for Scalar DB/Scalar DL deployment on AWS
+# Set up a database for ScalarDB/ScalarDL deployment on AWS
 
-This guide explains how to set up a database for Scalar DB/Scalar DL deployment on AWS.
+This guide explains how to set up a database for ScalarDB/ScalarDL deployment on AWS.
 
-## DynamoDB
+## Amazon DynamoDB
 
-Amazon DynamoDB is available for use in AWS by default. You do not need to set up anything manually to use it.
+### Authentication method
 
-### Optional steps
+When you use DynamoDB, you must set `REGION`, `ACCESS_KEY_ID`, and `SECRET_ACCESS_KEY` in the ScalarDB/ScalarDL properties file as follows.
 
-* Configure advanced monitoring services with the [AWS official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/monitoring-automated-manual.html), monitoring is enabled in DynamoDB by default.
-* You should change the `Read/Write Capacity` of DynamoDB tables based on your requirements.
+```properties
+scalar.db.contact_points=<REGION>
+scalar.db.username=<ACCESS_KEY_ID>
+scalar.db.password=<SECRET_ACCESS_KEY>
+scalar.db.storage=dynamo
+```
 
-Note:-
+Please refer to the following document for more details on the properties for DynamoDB.
 
-* Scalar DB/DL schema loader will configure the same Read/Write Capacity for all tables.
+* [Getting Started with ScalarDB on DynamoDB](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-dynamodb.md)
 
-## AWS RDS for MySQL, PostgreSQL, Oracle and SQL Server
+### Required configuration/steps
 
-### Steps
+DynamoDB is available for use in AWS by default. You do not need to set up anything manually to use it.
 
-* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) to create an AWS RDS MySQL/PostgreSQL/Oracle/SQL Server database instance.
-* (Optional) Configure advanced monitoring services with the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Monitoring.html), monitoring is enabled by default on AWS RDS database instances.
+### Optional configurations/steps
+
+#### Enable pint in time recovery (Recommended in the production environment)
+
+You can enable PITR as a backup/restore method for DynamoDB. If you use [ScalarDB Schema Loader](https://github.com/scalar-labs/scalardb/blob/master/docs/schema-loader.md) for creating schema, it enables the PITR feature for tables by default. Please refer to the official document for more details.
```

Comment:
I think it would be helpful if we could say why it should be enabled.
Also, it would be more user-friendly if we could add the following link somewhere.
https://github.com/scalar-labs/scalardb/blob/master/docs/backup-restore.md

The same comment applies to the sections of the other databases in the doc.